_data/publications.yml

```diff
@@ -82,3 +82,10 @@
   authors: "**, Jack B.** and Kevin M. Dalton, Doeke R. Hekstra"
   url: https://scripts.iucr.org/cgi-bin/paper?S160057672100755X
   info: Journal of Applied Crystallography, 2021
+- nickname: SFCalculator
+  section: cite
+  title: "SFCalculator: connecting deep generative models and crystallography"
+  first_author: "Minhuan Li"
+  authors: ", Kevin M. Dalton, Doeke R. Hekstra"
```

Comment:
this list. it's not an issue for this PR perhaps, but:
* the leading comma feels like a "gotcha"
* shouldn't it be a YAML list of strings, one entry per author, rather than a string with commas?

Reply:
Yeah it took me too long to figure out how to format this YAML. I agree this needs a refactor. I made an issue about it.